scripts/release.sh

```diff
@@ -0,0 +1,278 @@
+    echo "changing version in $pod_spec_file to $new_version_number"
+    # Swap new version in to file
+    sed '/s.version/{s/'$current_version'/'$new_version_number'/;}' $pod_spec_file > $pod_spec_new_file
+    mv -f $pod_spec_new_file $pod_spec_file
+fi
+
+# Step 2: Update RPC and protocol versions in /SmartDeviceLink/private/SDLGlobals.m
+echo 
+echo "Step 2: Checking SDLGlobals.m for RPC and Protocol versions"
+file="SmartDeviceLink/private/SDLGlobals.m"
+current_rpc_version=$(sed -n '/SDLMaxProxyProtocolVersion/{s/^.*@//;s/[\;]//;s/[\"]//g;p;q;}' $file)
+current_protocol_version=$(sed -n '/SDLMaxProxyRPCVersion/{s/^.*@//;s/[\;]//;s/[\"]//g;p;q;}' $file)
+echo "Current RPC Version: "$current_rpc_version
+echo "Current Protocol Version: "$current_protocol_version
+echo "If these are not correct, please update protocol versions in /SmartDeviceLink/private/SDLGlobals.m. Then press enter."
+read user_input
+
+# 4 Update to the newest BSON submodule. Update Package.swift and CocoaPods dependency files to point to latest if necessary.
+# extract version and link from Package.swift
+dependency_file="Package.swift"
+submodule_info="$(sed -n '/.package/{s/let package = Package(//;s/.package(//;s/)//;s/^[[:space:]]*//g;p;}' $dependency_file)"
+if [ ! -z "$submodule_info" ]; then
+    #loop through our dependencies
+    while IFS= read -r record ;
+    do
+        if [ ! -z "$record" ]; then
+            record_info="$(sed 's/,/\n/g' <<< $record)"
+            record_name=$(sed -n '/name:/{s/name://g;s/^[[:space:]]*//g;s/"//g;p;}' <<< "$record_info")
+            record_url=$(sed -n '/url:/{s/url://g;s/^[[:space:]]*//g;s/"//g;p;}' <<< "$record_info")
+            record_version=$(sed -n '/from:/{s/from://g;s/^[[:space:]]*//g;s/"//g;p;}' <<< "$record_info")
+            
+            # figure out latest version (visit link?)
+            #submodule_latest_version=$(gh repo view $record_url --json latestRelease -q .latestRelease.tagName)
+
+            # compare versions
+            #if [ $record_version != $submodule_latest_version ]; then
+                echo
+                echo "Current version of $record_name: "$record_version
+                #echo "Latest version of $record_name: "$submodule_latest_version
+            #fi
+        fi
+    done <<< "$submodule_info"
+    echo
+    echo "Please update the submodules before continuing with the release."
+    echo "You must also edit the dependancy information in SmartDeviceLink-iOS.podspec"
+    read user_input 
+fi
+
+# 5 update changelog
+# TODO - insert a template into the changelog that includes the version the users have selected above.
+#echo "A template for this release has been inserted into the changelog.  Please update it."
+echo 
+echo "Please update CHANGELOG.md, then return here and press enter."
+read user_input
+
+
+# 6 generate documentation
+echo 
+prompt_user "Would you like to automatically generate documentation with Jazzy?"
```

Comment:
Prompts currently have two question marks:

`Would you like to automatically generate documentation with Jazzy? (Y/N)?`

The first question mark should be removed throughout the file.